app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/DatabaseChangelog.java

```diff
@@ -1993,4 +1995,79 @@ public void changeAppLayoutTypeDefinition(MongoOperations mongoOperations, Mongo
         }
     }
 
+    @ChangeSet(order = "060", id = "update-mysql-postgres-mongo-ssl-mode", author = "")
+    public void updateMysqlPostgresMongoSslMode(MongoTemplate mongoTemplate) {
+        Plugin mysqlPlugin = mongoTemplate
+                .find(query(where("name").is("Mysql")), Plugin.class).get(0);
+
+        Plugin mongoPlugin = mongoTemplate
+                .find(query(where("name").is("MongoDB")), Plugin.class).get(0);
```

Comment:
Can we do the filtering by `packageName` instead of `name`? The name is supposed to be volatile, but the package name is supposed to be used for querying and is not intended to be changed.

Also, any reason we are fetching all and indexing at `0` instead of using `findOne`? There'll only be a single plugin that matches this query, so shouldn't make a difference anyway.

```suggestion
        Plugin mysqlPlugin = mongoTemplate
                .findOne(query(where("packageName").is("mysql-plugin")), Plugin.class);

        Plugin mongoPlugin = mongoTemplate
                .findOne(query(where("packageName").is("mongo-plugin")), Plugin.class);
```